spec/features/unauthenticated_user/unauthenticated_user_can_visit_homepage_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+describe "an unauthenticated user" do
+
+  it "can visit the home page" do
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.